### DISCUSSION TITLE
Support hardware video decoding out of the box! h265 included.

### DISCUSSION THREAD

**Discussion by adminy** [zen-browser/desktop#3188] [Ideas]
One thing firefox did more out of protest than anything else on linux was to not let you use your hardware like the h265 decoder that your GPU likely supports. They recently added it on windows (yes windows only) because the OS can pick up the licensing tab. On linux however its very easy to make ffmpeg handle all the hardware decoding. Its just not the default method, and firefox explicitly disables h265 in protest of course.

On Macos safari supports h265 out of the box, Also I suspect av-foundation api also has a way for avoididing all the licensing since its done in hardware and its paid by hardware on mac. 

Lets make video playback great again! We're sold the promise that AV1 will solve all of this while most hardware today is h265. Lets improve todays browsers, not tomorrow's promise.

* h265 -hevc is another name for it.

**Comment by adminy** [comment]
![image](https://github.com/user-attachments/assets/54a614c2-7979-4e1b-a949-fd2f3d88433a)

**Comment by stpnwf** [comment]
Yes, this! Please support HEVC out of the box in Zen browser; even if just a software decode. I know it is a "paid format", but it plays really well on VLC and MPV. I am guessing they are using a reverse engineered codec. So why not implement the same solution in Zen Browser? They are both open source projects.

**Comment by CritLoren** [comment]
I wanted to add to the requests for this, firefox already supports this since 137, but it looks like Zen doesn't yet?

**Comment by Melancholis** [comment]
still doesn't support yea?

**Reply by CritLoren** [reply]
That is correct
